Web3privacynowplatform/Briefs/NAMADA.md

```diff
@@ -0,0 +1,88 @@
+**Basic info**
+
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Project name**  | NAMADA |
+| **Logotype**  |  https://github.com/anoma/namada-docs/blob/master/packages/docs/public/images/namada-logo.svg |
+| **Short project description (1 sentence)** | NAMADA is a layer 1 blockchain solution that redefines asset-agnostic, multichain privacy. |
+| **Launch date** | TBD |
+| **Is it open source?** | YES |
+| **Website**  | https://namada.net/ |
+| **Github**  | https://github.com/anoma/namada |
+
+**Token**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Do you have a token** | NAM(in testnet) |
+
+**Socials**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Blog** | https://namada.net/blog |
+| **Twitter (X)** | https://twitter.com/namada |
+| **Telegram** | https://t.me/namadaprotocol  |
+| **Discord** | https://discord.com/invite/namada |
+| **Other** | https://specs.namada.net/ |
+
+**Docs**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Docs** | https://docs.namada.net/ |
+
+**Team**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Is your team anon or public**  | public | 
+| **List core contributors** | https://github.com/orgs/anoma/people | 
+
+**Funding**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **How are you funded**  | Mixed | 
+| **List VCs backing you**  | Delphi Digital, Dialectic, KR1, Spartan, NGC, MH Ventures, Bixin Ventures, No Limit, WAGMI, Plassa, Perridon Ventures, Anagram,  Factor |
+| **List VCs round**  | Seed, Series: A,C | 
+| **Raised money** | https://www.crunchbase.com/organization/anoma | 
+
+**Github**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **What licence is in use**  | GPL-3.0 |
+
+**Technology**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Technical specialisation**  | Asset-agnostic, interchain privacy | 
+| **Specify technology readiness**  | testnet | 
+| **Features**  | Shielded Swaps, Multi-Asset Shielded Pool, IBC, Trust-minimized 2-way Ethereum bridge | 
+| **Encryption method**  | TODO |
+
+**Privacy**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Is it peer-to-peer**  | YES | 
+| **Complience**  | NO |
+| **KYC**  | NO |
+| **Privacy policy** | - |
+| **Collected data**  | NO |
+| **Data sharing** | NO |
+| **Sign-in requirments** | NO |
+| **Identity integrations** | NO |
```

Comment:
Not sure about it. Some points might be marked TBD (To be determined) because NAMDA is in its early stages, and this information could change over time.